libsrc/stdio/tvc/fputc_cons.asm

```diff
@@ -0,0 +1,28 @@
+;
+;   Videoton TV Computer C stub
+;   Sandor Vass - 2019
+;   Based on the source of
+;
+;	Enterprise 64/128 C Library
+;
+;	Fputc_cons
+;
+;	Stefano Bodrato - 2011
+;
+
+	SECTION code_clib
+	PUBLIC  fputc_cons_native
+    INCLUDE "target/tvc/def/tvc.def"
+
+;
+; Entry:        stack contains a char
+;
+.fputc_cons_native
+    ld      hl,2
+    add     hl,sp
+    ld      a,(hl)
```

Comment:
Does the value need to be in a and c? Just use `ld c,(hl)`?

Reply:
A simple copy-paste bug. Enough to have it in C, I will fix it.

Reply:
if you commit on the branch from where this PR was created, it will get incrementally added to this PR (as far as I can tell how it works for me on other github projects, didn't try PR for z88dk yet)